napari/utils/indexing.py

```diff
@@ -4,6 +4,15 @@
 import numpy.typing as npt
 
 
+def visible_items_in_slice(
+    index: Tuple[npt.NDArray[np.int_], ...], position_in_axes: Dict[int, int]
+) -> npt.NDArray[np.bool_]:
+    queries = [
+        index[ax] == position for ax, position in position_in_axes.items()
+    ]
+    return np.logical_and.reduce(queries, axis=0)
```

Comment:
Can you add a docstring here? I don't understand what this function is supposed to do. (And I'm not sure it should be public...?)

Reply:
Added. Should I make it private? 

The question is why the whole module is public.

Reply:
Sure, but given that it is, let's not expand it.

Reply:
moved